data/coin/goldCoin.png.data

```diff
@@ -0,0 +1,23 @@
+texture: coin/goldCoin
+dimensions: 1, 1
+sprites: 5
+
+clip: 0, 0, 16, 16
+pivot: 0.5, 0.5
+scale: 1, 1
+
+clip: 16, 0, 16, 16
+pivot: 0.5, 0.5
+scale: 1, 1
+
+clip: 32, 0, 16, 16
+pivot: 0.5, 0.5
+scale: 1, 1
+
+clip: 48, 0, 16, 16
+pivot: 0.5, 0.5
+scale: 1, 1
+
+clip: 64, 0, 16, 16
+pivot: 0.5, 0.5
+scale: 1, 1
```

Comment:
No new line at the end of file as GitHub also noted.
Integrating #30 and saving the files again should solve this.

Reply:
This comment still applies.